src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -97,6 +111,9 @@ class RuleConditionsForm extends React.PureComponent<Props, State> {
     );
     environmentList.unshift([null, anyEnvironmentLabel]);
 
+    const fieldOptions = generateFieldOptions(this.props.organization);
```

Comment:
We have a local `organization` 

```suggestion
    const fieldOptions = generateFieldOptions(organization);
```